tests/codegen/simd/packed-simd.rs

```diff
@@ -51,7 +51,7 @@ pub fn square_packed_full(x: Simd<f32, 3>) -> FullSimd<f32, 3> {
 pub fn square_packed(x: Simd<f32, 3>) -> Simd<f32, 3> {
     // CHECK-NEXT: start
     // CHECK-NEXT: load <3 x float>
-    // noopt-NEXT: load <3 x float>
+    // CHECK-NOOPT-NEXT: load <3 x float>
```

Comment:
This was made slightly harder to read. The rev name was sorta intentional, to make it stand out more, but now it blurs into the other CHECK*NEXTs and becomes easier to overlook. maybe it should be `"{rev}-CHECK"`?

Reply:
hm, I wonder if FileCheck would accept `[{REV}]-CHECK`?

Reply:
> hm, I wonder if FileCheck would accept `[{REV}]-CHECK`?

Any string similar to a variable name is ok.